zebra/rt_netlink.c

```diff
@@ -1535,6 +1535,12 @@ static int netlink_route_multipath(int cmd, struct zebra_dplane_ctx *ctx)
 		char buf[NL_PKT_BUF_SIZE];
 	} req;
 
+	if (zrouter.skip_kernel_install) {
```

Comment:
the question about netlink-only aside, you've chosen only to block route installation. what about the other things that zebra installs - like ... LSPs? are you going to ask to add another command-line letter for each of those install types?

Reply:
This is valid comment as it is already in my mind. I was planning a separate commandline flag for MPLS labels.

Reply:
I think of, the same should be applied to MAC updates coming from BGP ?

Reply:
so ... trying to have some fine-grained setting on the command-line seems really awkward. I really think that if you want to have the level of control that you seem to want, you should invest in a simple dataplane extension module.